resources/assets/js/components/DetailsEditable.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <form onsubmit="return false">
+        <div class="form-group">
+            <label for="title" class="mb-0">Title</label>
+            <input type="text" name="title" id="title' + indexItem" v-model="localTitle" @input="change" placeholder="Title" :disabled="!editing"
+                :class="{'form-control': editing, 'form-control-plaintext': !editing }">
+            <label for="description" class="mb-0 mt-2">Description</label>
+            <textarea id="description" cols="30" v-model="localDescription" @input="change" placeholder="Description" :disabled="!editing"
+                :class="{'form-control': editing, 'form-control-plaintext': !editing }" :rows="numberOfRows"></textarea>
+            <input type="image" class="buttonIcon right" :src="getImage" @click="editing = !editing" alt="Edit">
+        </div>
+    </form>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      editing: false,
+      localTitle: "",
+      localDescription: ""
+    };
+  },
+
+  props: {
+    title: {
+      type: String,
+      required: true,
+      default: () => {
+        return "";
+      }
+    },
+    description: {
+      type: String,
+      required: true,
+      default: () => {
+        return "";
+      }
+    },
+    numberOfRows: {
+      type: String,
+      required: false,
+      default: () => {
+        return "3";
+      }
+    }
+  },
+
+  methods: {
+    change() {
+      this.$emit("change", {
+        title: this.localTitle,
+        description: this.localDescription
+      });
+    },
+    refresh() {
+      this.localTitle = this.title;
+      this.localDescription = this.description;
+    }
+  },
+
+  computed: {
+    getImage: function() {
+      if (this.editing) return "/images/check.svg";
+      else return "/images/pencil.svg";
+    }
+  },
+
+  mounted: function() {
+    this.refresh();
+  },
+
+  watch: {
+    title: function() {
+      this.refresh();
+    },
+    localTitle: function() {
+      this.refresh();
+    }
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+.right {
```

Comment:
Bootstrap already has such a class for you

Reply:
Yup, I found that out later as well, thanks for pointing this missed one out.